examples/fit_fake_data.py

```diff
@@ -23,6 +23,6 @@
 with figlist_var() as fl: 
     fl.next('fit with guess')
     fl.plot(fake_data,'o',label='fake data')
-    fl.plot(f,label='fit')
+    fl.plot(f.eval(100),label='fit')
     fl.plot(guess,label='guess')
```

Comment:
Your code was just plotting the data -- I think it's ready to go, but I want to make sure you see this change.

Reply:
okay so I was confused on why the eval is necessary. It seems by looking through core.py eval applies the actual newdata fit...is this correct?

Reply:
eval doesn't really apply anything.  It calculates the fit function along the coordinate axis of `f`, in this case.  The 100 tells it to use 100 evenly space points, rather than just using the coordinate axis as-is.  So, before merging this, I'm going to ask that you edit the docstring of eval so that you understand what's going on.

Reply:
done